resources/aws_ec2.go

```diff
@@ -374,19 +383,28 @@ func (obj *AwsEc2Res) Watch() error {
 	if obj.WatchListenAddr != "" {
 		return obj.snsWatch()
 	}
-	return obj.longpollWatch()
+	return obj.waiterWatch()
 }
 
-// longpollWatch uses the ec2 api's built in methods to watch ec2 resource state.
-func (obj *AwsEc2Res) longpollWatch() error {
-	send := false
+// waiterWatch uses the ec2 api's built in methods to watch ec2 resource state.
+func (obj *AwsEc2Res) waiterWatch() error {
+	var send bool
```

Comment:
why are we renaming this and also we are we making these sort of unrelated changes in the same commit?

`send := false`
is functionally the same as
`var send bool`

Reply:
I can leave it as longpollWatch, but IMO, longpolling and using the waiters aren't exactly the same, and I think it's causing some confusion. I can also change it back to `send := false`, (I just like `var send bool` better style wise and thought it was okay since this is a refactor)